opsdroid/logging.py

```diff
@@ -80,13 +80,14 @@ def configure_logging(config):
 
     rootlogger.setLevel(log_level)
 
+    formatter = logging.Formatter("%(levelname)s %(name)s: %(message)s")
     try:
```

Comment:
I'd suggest this to be changed to something like this to avoid using the try/except: pass.

```suggestion
    with contextlib.suppress(KeyError):
        if config["logging"]["extended"]:
             formatter = logging.Formatter(
                 "%(levelname)s %(name)s.%(funcName)s(): %(message)s"
             )
```

With contextlib.suppress, the KeyError gets suppressed and will automatically pass.